client/src/components/VoterRequirements.jsx

```diff
@@ -1,69 +1,175 @@
-    <p>If you’re not a first-time voter, you may have different questions - hopefully we can help! </p>
-    <h3 className="VoterRequirements-heading">Changing Registration Information</h3>
-    <p>The North Carolina Voter Registration Application may be used to change any voting information, including: name, address and party affiliation. The change notification must be signed, and should be sent to the appropriate County Board of Elections by the voter registration deadline (25 days before each election).</p>
-    <p>Once you have completed and mailed your changes to your local board of elections, allow two to six weeks for delivery of your voter identification card. Read the information thoroughly and note any changes or mistakes on the card. A voter identification card is for your use only, you do not need the card in order to vote.</p>
-    <h3 className="VoterRequirements-heading">Military & Overseas Citizens Voting Information</h3>
-    <p>Some citizens, such as people serving in the military who are away from their permanent home on Election Day, their families, or U.S. citizens who are living abroad, have special voting rights and ways to register to vote. </p>
-    <p>If this describes you, you have the choice of either requesting a mail-in absentee ballot the same way as other registered voters, or applying to register and/or vote through special programs for military and overseas voters as described here.</p>
-    <h3><a href="https://ncvoter.org/registering-to-vote/">For more information visit this page from Democracy NC</a></h3>
+    <p>
+      If you’re not a first-time voter, you may have different questions -
+      hopefully we can help!{" "}
+    </p>
+    <h3 className="VoterRequirements-heading">
+      Changing Registration Information
+    </h3>
+    <p>
+      The North Carolina Voter Registration Application may be used to change
+      any voting information, including: name, address and party affiliation.
+      The change notification must be signed, and should be sent to the
+      appropriate County Board of Elections by the voter registration deadline
+      (25 days before each election).
+    </p>
+    <p>
+      Once you have completed and mailed your changes to your local board of
+      elections, allow two to six weeks for delivery of your voter
+      identification card. Read the information thoroughly and note any changes
+      or mistakes on the card. A voter identification card is for your use only,
+      you do not need the card in order to vote.
+    </p>
+    <h3 className="VoterRequirements-heading">
+      Military & Overseas Citizens Voting Information
+    </h3>
+    <p>
+      Some citizens, such as people serving in the military who are away from
+      their permanent home on Election Day, their families, or U.S. citizens who
+      are living abroad, have special voting rights and ways to register to
+      vote.{" "}
```

Comment:
Same comment about `{" "}`

Reply:
The {" "} is one of two methods for adding space between elements in html inside a react JSX file - it strips out white space when it renders the HTML in places where you have other elements like embedded anchor tags. They were added by the linter, not coded in by hand. (&nbsp; is the other and there's issues with using HTML entities in JSX as well so the first approach is generally preferred at least as I understand it)